src/Common/Runner/Nuget/NuGetPackageResolver.cs

```diff
@@ -20,38 +23,45 @@ public NuGetPackageResolver(string packagesStore, string nugetFeed = Constants.D
         public async Task<Package> DownloadPackageWithDependencies(PackageInfo packageInfo)
         {
             var latesVersion = await GetLatestPackageVersion(packageInfo);
-            var version = SemVer.Parse(latesVersion);
 
-            var packageLocation = await DownloadPackage(packageInfo, version);
-            var dependencies = await GetAllDependencies(packageInfo, version);
-
-            var files = NuGetV2.GetLibFiles(packageLocation).Concat(dependencies);
+            var packageLocation = await DownloadPackage(packageInfo, latesVersion);
+            var dependencies = await GetAllDependencies(packageInfo, latesVersion);
 
             var package = new Package
             {
                 Info = packageInfo,
-                Version = latesVersion,
-                Files = files.Where(f => Path.GetExtension(f.Name) == ".dll").ToArray()
+                Version = latesVersion.ToString(),
+                Files = dependencies.Where(f => Path.GetExtension(f.Name) == ".dll").ToArray()
```

Comment:
is that the same as the files contained before? it looks like files previously was only referring to files of a specific package while at first glance, it now looks like it contains fails of dependencies too?